aggregations/aggregations/library.whizzml

```diff
@@ -0,0 +1,242 @@
+(define UPPER_TO_LOWER {"A" "a" "B" "b" "C" "c" "D" "d" "E" "e"
+  "F" "f" "G" "g" "H" "h" "I" "i" "J" "j" "K" "k" "L" "l" "M" "m" "N" "n"
+  "O" "o" "P" "p" "Q" "q" "R" "r" "S" "s" "T" "T" "U" "u" "V" "v" "W" "w" "X"
+  "x" "Y" "y" "Z" "z"})
```

Comment:
whizzml has now lower-case, upper-case and capitalize, so you can probably simplify this code